docs/pages/patterns/combobox-input.md

```diff
@@ -288,15 +332,16 @@ To add text in the `body` of the component, simply include your text in the `fd-
                 <span class="fd-list__title">Kiwi</span>
             </li>
         </ul>
+    </div>
   </div>
 </div>
 <span class="fd-form-message fd-form-message--static fd-form-message--success">Success message</span>
 {% endcapture %}
 {% include display-component.html component=semantic %}
 
 ## Combobox Mobile Mode
-For mobile devices, or tablets, combobox component should be displayed in fullscreen mode.
-So instead of using popover and dropdown, it should be wrapped in `dialog` and `bar` components.
+For mobile devices or tablets, the Combobox component should be displayed in fullscreen mode.
+Instead of using popover and dropdown, it should be wrapped in `dialog` and `bar` components.
```

Comment:
popover and dropdown are components too. Should we capitalize them too as it was done in the previous line?

Reply:
not part of the scope of this PR but i will do the first one